src/components/bookItems.js

```diff
@@ -1,28 +1,52 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/booksSlice';
+import { removeBook, deleteBook } from '../redux/books/booksSlice';
 
-const BookItem = ({ book }) => {
+const BookItem = ({ book, id }) => {
   const dispatch = useDispatch();
 
   return (
-    <div className="book-item">
-
-      <div className="book-title">{JSON.parse(book).title}</div>
-      <div className="book-author">{JSON.parse(book).author}</div>
-      <button
-        type="button"
-        className="book-remove"
-        id={JSON.parse(book).id}
-        onClick={(evet) => dispatch(removeBook(evet.target.id))}
-      >
-        Remove Book
-      </button>
+    <div className="book-wrapper">
+      <div className="book-datail">
+        <div className="book-category">{JSON.parse(book).category}</div>
+        <div className="book-title">{JSON.parse(book).title}</div>
+        <div className="book-author">{JSON.parse(book).author}</div>
+        <div className="bool-detail-btn">
+          <button type="button" className="book-comment">Comment</button>
+          <button
```

Comment:
- [ ] The book prop you passed into the BookItem component is seen as being undefined. As a result there is an error that's preventing the app from loading on the browser. Kindly take a look and fix.

### Error on the browser
![error](https://user-images.githubusercontent.com/26597432/232188528-6c76f2f8-3ad6-442c-81e1-754c9a19b56f.png)